scalability_and_performance/index.adoc

```diff
@@ -0,0 +1,61 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="index"]
+= {product-title} scalability and performance overview
+include::_attributes/common-attributes.adoc[]
+:context: index
+
+// ifndef::openshift-dedicated,openshift-rosa[]
+{product-title} offers best practices and utilities to help you optimize the performance and scale of your {product-title} clusters. The following documentation provides information on recommended performance and scalability practices, reference design specifications, optimization and low latency tuning.
+
+To engage with Red Hat support, see xref:../support/getting-support.adoc#getting-support[Getting support].
```

Comment:
Why "engage" over "contact" or similar?

Reply:
That was my suggestion - after a quick sweep of the repo, it seems I'm on the only one that uses that language :) probably an artifact from my time working in support years ago.

"Contact" is fine too, of course!

Reply:
Done, thank you.